edvaudin.ExcelReader/DbFactory.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using System.Text;
+
+namespace edvaudin.ExcelReader;
+
+internal class DbFactory
+{
+    private readonly string connectionString = ConfigurationManager.ConnectionStrings["exceldb"].ConnectionString;
+    public void CreateTable(DataTable table)
+    {
+        using var conn = new SqlConnection(connectionString);
+        conn.Open();
+        string sql = GetCreateTableQuery(table);
+        SqlCommand cmd = new(sql, conn);
+        cmd.ExecuteNonQuery();
+    }
+
+    public void DropExistingDataBase()
+    {
+        using var conn = new SqlConnection(connectionString);
+        conn.Open();
+        string sql = "DROP DATABASE exceldb";
```

Comment:
Project is looking great!! 

Tiny thing.... Looks like you'll need to check for the existence of the DB first:

![image](https://user-images.githubusercontent.com/108711476/218566763-9dc1a748-a161-49be-84d5-38afe0ce30b4.png)